omnigibson/tasks/task_base.py

```diff
@@ -116,6 +117,10 @@ def load(self, env):
         # Run internal method
         self._load(env=env)
 
+        # Load the obs space dim
+        obs = self.get_obs(env=env, flatten_low_dim=True)
```

Comment:
should this return `obs, info` 2-tuple to unify with all other `get_obs()` calls?

Reply:
+1